pkg/reconciler/v1alpha1/taskrun/resources/output_resource_test.go

```diff
@@ -236,6 +236,45 @@ func Test_Valid_OutputResources(t *testing.T) {
 			}},
 		}},
 		build: build(),
+	}, {
+		name: "image resource in output with pipelinerun with owner",
+		desc: "image resource declared as output with pipelinerun owner reference should not generate any steps",
+		taskRun: &v1alpha1.TaskRun{
```

Comment:
nit: maybe having `wantedSteps` and `wantedVolumes` explicitely `nil` or empty ; just to help knowing what we are "expecting" when reading the case.

Reply:
Thats a good point. I have updated that